scrapers/Xevunleashed.yml

```
@@ -0,0 +1,34 @@
+name: "XevUnleashed"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - xevunleashed.com/updates/
+    scraper: sceneScraper
+
+
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //span[@class="update_title"]/text()
+      Date:
+        selector: //span[@class="availdate"]/text()
+        concat: " "
+        postProcess:
+          - replace:
+              - regex: (\d{2}\/\d{2}\/\d{4}).*
+                with: $1
+          - parseDate: 01/02/2006
+      Details: //meta[@name="description"]/@content
```

For long descriptions this only get's us partial meta.
```suggestion
      Details: //span[@class="latest_update_description"]
```
seems to work ok